js/react/pages/admin/wishcardsAdministrationPage.jsx

```diff
@@ -0,0 +1,142 @@
+import axios from 'axios';
+import React, { useState, useEffect } from 'react';
+
+export default function WishcardsAdministrationPage() {
+	const [agenciesWithWishCards, setAgenciesWithWishCards] = useState(new Object());
+
+	useEffect(() => {
+		fetcAgencyWithhWishCardsData(setAgenciesWithWishCards);
+	}, []);
+
+	const handleUpdateWishcard = (wishCardId, wishItemUrl) => {
+		axios
+			.put(`/api/admin/`, { wishCardId, wishItemUrl })
+			.then(() => {
+				fetcAgencyWithhWishCardsData(setAgenciesWithWishCards);
+			})
+			.catch((error) => {
+				window.showToast(
```

Comment:
we do not have toastify anymore, instead you can use

```js
new window.DG.Toast().show(
	error?.response?.data?.error?.msg || error?.message || 'Unable to publish wishcard',
	true
);
```

code can be found in `toast.js`